1.0.0 (#38)
searis/node-red-contrib-clarify: README.md

![Clarify Insert Node](https://github.com/searis/node-red-contrib-clarify/blob/master/examples/random-data-example.png?raw=true)

Any questions? Send us an email on support@clarify.us

## Changelog

> From 1.0.0 we will not do breaking changes to the overall msg.format. 

Reviewer: ```suggestion
From v1.0.0-beta4 we do not expect to do any breaking changes to the input message format. However we might add new optional fields. As for the block output, development is still ongoing, and breaking changes must be expected before the final v1.0.0 release.
```

> ## Changelog
> 
> From 1.0.0 we will not do breaking changes to the overall msg.format. 

Reviewer: ```suggestion
## Backwards compatibility

From v1.0.0-beta.4 we do not expect to do any breaking changes to the insert node input message format. However we might add new optional fields.

As for the insert node _output_ message format, development is still ongoing, and breaking changes must be expected before the final v1.0.0 release. The number of outputs may also change.

## Changelog

The changelog is introduced from `v.1.0.0-beta.4`, and describe changes from  `v.1.0.0-beta.3`.

```

> ### 1.0.0 - Breaking changes from -beta.X

Reviewer: Nice change log.

But should we do another beta?

I don't think we want to do any more changes to the input structure, but I don't think we have yet nailed the output yet.

Reviewer: #37

Updated the format of the messages according to this proposal: https://github.com/searis/node-red-contrib-clarify/issues/28

- The Input ID is put in `msg.topic`
- The signal meta data is moved out of the payload to `msg.signal`

> - `msg.payload.series` is renamed to `msg.payload.values`

Reviewer: Wasn't it inside a `data` sub-structure before?
```suggestion
- `msg.payload.data.times` is renamed/moved to `msg.payload.times`
- `msg.payload.data.series` is renamed/moved to `msg.payload.values`
```

New message format:

> ```json

Reviewer: Try `js`

````suggestion
```js
````

msg:
  topic: "<Input ID>"
  payload:
    times: ["<timestamp>", ...]
    values: [(<number>||null), ...]

>   signal: <Signal> # Match https://docs.clarify.us/reference#signal

Reviewer: ```suggestion
  signal: <Signal> // Match https://docs.clarify.us/reference#signal
```
